bcda/main.go

```diff
@@ -379,3 +401,49 @@ func getEnvInt(varName string, defaultVal int) int {
 	}
 	return defaultVal
 }
+
+func removeExpired(hrThreshold int) {
+	fmt.Println("Cleaning up expired jobs...")
+	db := database.GetGORMDbConnection()
+	defer db.Close()
+
+	var jobs []models.Job
+	err := db.Find(&jobs, "status = ?", "Completed").Error
+	if err != nil {
+		log.Error(err)
+	}
+
+	expDir := os.Getenv("FHIR_EXPIRED_DIR")
+	if _, err = os.Stat(expDir); os.IsNotExist(err) {
+		err = os.MkdirAll(expDir, os.ModePerm)
+		if err != nil {
+			log.Error(err)
+		}
+	}
+
+	for _, j := range jobs {
+		t := j.CreatedAt
+		elapsed := time.Since(t).Hours()
+		if int(elapsed) >= hrThreshold {
+			j.Status = "Expired"
+			err = db.Save(j).Error
+			if err != nil {
+				log.Error(err)
+			}
+
+			id := strconv.Itoa(int(j.ID))
```

Comment:
this seems complicated.  Why not just do string(j.ID)?  or jobDir := fmt.Sprintf("%s/%d", os.Getenv("FHIR_PAYLOAD_DIR"), j.ID) works as well.

Reply:
Neither one of those options will work.